TTS/auto_tts/complete_recipes.py

```diff
@@ -0,0 +1,159 @@
+from TTS.auto_tts.model_hub import TtsModels, VocoderModels
+from TTS.auto_tts.utils import data_loader
+from TTS.trainer import Trainer, TrainingArgs, init_training
+
+
+class TtsExamples:
+    """This is trainer for calling complete recipes based off public datasets.
+    all configs are based off pretrained model configs or the model papers.
+
+    usage:
+            From TTS.auto_tts.complete_recipes import TtsExamples
+            trainer = TtsExamples(data_path='DEFINE THIS', batch_size=32, learning_rate=0.001,
+                      mixed_precision=False, output_path='DEFINE THIS', epochs=1000)
+            model = trainer.ljspeechAutoTts("tacotron2, "double decoder consistency")
+            model.fit()
+    """
+
+    def __init__(self, data_path, batch_size, output_path, mixed_precision, learning_rate, epochs):
+
+        self.data_path = data_path
+        self.batch_size = batch_size
+        self.output_path = output_path
+        self.mixed_precision = mixed_precision
+        self.learning_rate = learning_rate
+        self.epochs = epochs
+        self.model = TtsModels(
+            batch_size=self.batch_size,
+            mixed_precision=self.mixed_precision,
+            learning_rate=self.learning_rate,
+            epochs=self.epochs,
+        )
+
+    def ljspeechAutoTts(
```

Comment:
Better to use `_` notation instead of Camel notation to comply with the rest of the code base.